.github/actions/image-build/action.yml

```diff
@@ -0,0 +1,57 @@
+# Copyright (C) 2024 Intel Corporation
```

Comment:
Is it possible to put actions/image-build/action.yml to Validate repo? Then all repo can leverage this action.